testutil/testutil_test.go

```diff
@@ -0,0 +1,30 @@
+package testutil
+
+import (
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"testing"
+)
+
+func TestDescriptorFromProtoSource_CustomProtoPaths(t *testing.T) {
```

Comment:
Can you add a test for the `descriptor_set_in`?

Reply:
Done.